tests/protocol/test_hostplugin.py

```diff
@@ -473,29 +473,41 @@ def test_validate_block_blob(self):
 
         exp_method = 'PUT'
         exp_url = hostplugin_status_url
-        exp_data = self._hostplugin_data(
-            status_blob.get_block_blob_headers(len(faux_status)),
-            bytearray(faux_status, encoding='utf-8'))
 
-        with patch.object(restutil, "http_request") as patch_http:
-            patch_http.return_value = Mock(status=httpclient.OK)
+        original_strftime = time.strftime
```

Comment:
but aren't all the tests that call _validate_hostplugin_args affected by the same issue?

Reply:
instead of mocking the formatting function, you could relax the comparison in _compare_data for this specific header

Reply:
You're correct, I was focusing on a single test that failed. I've updated the tests by removing the seconds from the timestamps when comparing in `_compare_data`.